dietpi/dietpi-software

```diff
@@ -12119,7 +12119,13 @@ _EOF_
 			fi
 			G_EXEC rm -Rf "$ha_home/.homeassistant"
 			G_EXEC ln -s /mnt/dietpi_userdata/homeassistant "$ha_home/.homeassistant"
+
+			# Download HACS
+			Download_Install 'https://github.com/hacs/integration/releases/latest/download/hacs.zip' hacs
+			[[ -d '/mnt/dietpi_userdata/homeassistant/custom_components/hacs' ]] && G_EXEC rm -Rf /mnt/dietpi_userdata/homeassistant/custom_components/hacs
```

Comment:
`-f` flag means to ignore when file/dir is missing, but we explicitly check whether it exists:
```suggestion
			[[ -d '/mnt/dietpi_userdata/homeassistant/custom_components/hacs' ]] && G_EXEC rm -R /mnt/dietpi_userdata/homeassistant/custom_components/hacs
```